library/src/main/java/com/bumptech/glide/Glide.java

```diff
@@ -145,24 +147,97 @@ public static Glide get(Context context) {
     if (glide == null) {
       synchronized (Glide.class) {
         if (glide == null) {
-          Context applicationContext = context.getApplicationContext();
-          List<GlideModule> modules = new ManifestParser(applicationContext).parse();
-
-          GlideBuilder builder = new GlideBuilder(applicationContext);
-          for (GlideModule module : modules) {
-            module.applyOptions(applicationContext, builder);
-          }
-          glide = builder.createGlide();
-          for (GlideModule module : modules) {
-            module.registerComponents(applicationContext, glide.registry);
-          }
+          glide = initGlide(context);
         }
       }
     }
 
     return glide;
   }
 
+  private static Glide initGlide(Context context) {
+    Context applicationContext = context.getApplicationContext();
+    List<GlideModule> manifestModules = new ManifestParser(applicationContext).parse();
```

Comment:
Just a reminder to ensure there is a way to totally disable manifest parsing in keeping with #1754   Thanks!